src/Tempest/Generation/tests/ClassGeneratorTest.php

```diff
@@ -50,8 +50,8 @@ public function creates_methods_with_parameters(): void
             //
         PHP, parameters: ['id' => 'int'], returnType: '?App\\Models\\User');
 
-        $this->assertMatchesSnapshot($class->print());
-    }
+        $class->simplifyImplements();
+        $class->addImplement(FakeMigration::class);
```

Comment:
I think it's a mistake from my own.
To work with your updated `ClassManipulator` i've merged your waiting PR fork and then rebase on the main this day, I think i've done something wrong because I didn't change any of your files.

I'll fix that later, thanks for notice it 🙏

Reply:
I can't find a real version of this file, the entire method has no assertions so it break my tests, I've commented it for now.
Maybe an unmerged work 🤷‍♂️